lib/diagnostic.js

```diff
@@ -28,7 +34,12 @@
 // Attributes: name, address, numberOfPets
 
 // Replace null with your answer
-let owner = null;
+let owner = {
+          this.name : 'Rich Chunderson';
+          this.address : '1362 Alf Dr.';
+          this.numberOfPets : 28;
+
+}
 
```

Comment:
Looking for let owner = {
  name: 'Brian',
  address: '41 West St.',
  numberOfPets: 2
};